src/legal-pages/index.tsx

```diff
@@ -321,8 +321,7 @@ async function fetchContent<A extends Page>(
   const response = await fetchWithCache(page.url)
 
   if (response.ok) {
-    const text = await response.text()
-    const rawContent = page.url.endsWith('.md') ? markdownToHtml(text) : text
+    const rawContent = markdownToHtml(await response.text())
```

Comment:
Why do we now handle all legal-pages as markdown files?

Reply:
Because all legal pages we have and we will have will be in markdown format.